packages/react/src/hooks/useKeyPress.ts

```diff
@@ -6,7 +6,16 @@ type PressedKeys = Set<string>;
 type KeyOrCode = 'key' | 'code';
 
 export type UseKeyPressOptions = {
+  /**
+   * You may want to listen to key presses on a specific element. This field lets you configure
+   * that!
```

Comment:
copied from https://reactflow.dev/api-reference/hooks/use-key-press#options-target

I think we can simplify?

```suggestion
   * Listen to key presses on a specific element.
```